utils/src/server/rpc_module.rs

```diff
@@ -574,4 +588,61 @@ mod tests {
 		assert!(module.method("hello_world").is_some());
 		assert!(module.method("hello_foobar").is_some());
 	}
+
+	#[tokio::test]
+	async fn calling_method_without_server() {
+		// Call sync method with no params
+		let mut module = RpcModule::new(());
+		module.register_method("boo", |_: RpcParams, _| Ok(String::from("boo!"))).unwrap();
+		let result = &module.call_with("boo", &None::<()>).await.unwrap();
```

Comment:
dq: why take a reference here? Wouldn't `module.call_with("boo", &None::<()>).await.unwrap();` work?

Reply:
do you mean `module.call_with("boo", None::<()>).await.unwrap();`?

I think I messed up the trait bounds for that to work.